hr_employee_firstname/models/hr_employee.py

```diff
@@ -168,7 +168,6 @@ def _install_employee_firstname(self):
     def _update_partner_firstname(self):
         for employee in self:
             partners = employee.mapped("user_id.partner_id")
-            partners |= employee.mapped("address_home_id")
```

Comment:
Shouldn't this be replaced by `work_contact_id`? Or is it expected for it to be removed?